src/coreclr/jit/codegenwasm.cpp

```diff
@@ -16,7 +17,48 @@ void CodeGen::genMarkLabelsForCodegen()
 
 void CodeGen::genFnProlog()
 {
-    NYI_WASM("Uncomment CodeGen::genFnProlog and proceed from there");
+    ScopedSetVariable<bool> _setGeneratingProlog(&compiler->compGeneratingProlog, true);
```

Comment:
This should instead follow the NYI comment and be a partially ifdefed common `genFnProlog`.

(It can be a fully ifdefed one out for now if that works...)

Reply:
Ok, I folded it back together for now. Not sure I like this as a long term plan, but it's more a critique of the existing code than anything else.